feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6750697ee8b87f5e665c0712.md

---
id: 6750697ee8b87f5e665c0712
title: Task 41
challengeType: 22
dashedName: task-41
---

<!-- (Audio) Lisa: Perfect. Let's make sure we double check credentials after security updates in the future. -->

# --instructions--

Listen to the audio and complete the sentence below.

# --fillInTheBlank--

## --sentence--

`Perfect. Let's make sure we BLANK BLANK credentials after security updates in the future.`

## --blanks--

`double`

### --feedback--

This word means to do something twice or with extra care.

---

`check`

### --feedback--

This word refers to verifying or confirming something.

# --explanation--

> The phrase `double check` means to verify something a second time to ensure it is correct or accurate. It is often used when dealing with important tasks to avoid errors. Lisa suggests that they double check the credentials after security updates to ensure no mistakes are made in the future. Example: `We need to double check the calculations before submitting the report.` (meaning you'd like to verify if the calculations are right once more before sending the report to those who will analyze it)

Reviewer: ```suggestion
`Double check` means to verify something a second time to ensure it is correct or accurate. It is often used when dealing with important tasks to avoid errors.  For example: 

`We need to double check the calculations before submitting the report.` - Meaning you'd like to verify if the calculations are right once more before sending the report to those who will analyze it.
```
